packages/discount-code-exporter/package.json

```diff
@@ -20,7 +20,25 @@
     "Williams Omayuku <williams.omayuku@commercetools.com>"
   ],
   "main": "lib/index.js",
+  "bin": {
+    "discount-code-exporter": "bin/discount-code-exporter.js"
```

Comment:
Can we make the executable command one word

`discountcodeexporter` or shorter if possible